src/cpp/ext/filters/census/client_filter.cc

```diff
@@ -288,6 +288,15 @@ void OpenCensusCallTracer::OpenCensusCallAttemptTracer::RecordAnnotation(
   context_.AddSpanAnnotation(annotation, {});
 }
 
+void OpenCensusCallTracer::OpenCensusCallAttemptTracer::RecordAnnotation(
+    const Annotation& annotation) {
+  switch (annotation.type()) {
+    default:
+      // If tracing is disabled, the following will be a no-op.
+      context_.AddSpanAnnotation(annotation.ToString(), {});
```

Comment:
hmmm there will be some cost here (the cost of calling `ToString` and then throwing it away) - maybe we want to lift that check to the top of this function?

Reply:
Yeah I debated doing that but had left it as is since that was what the original RecordAnnotation function did. I changed both now.